app/views/application/_flash.html.slim

```diff
@@ -1,8 +1,9 @@
-#flash-message.container
-  - ApplicationController._flash_types.each do |severity|
-    - next unless flash[severity].present?
+.container
+  #flash-message
+    - ApplicationController._flash_types.each do |severity|
+      - next unless flash[severity].present?
 
-    - flash_mapping = {alert: :warning, notice: :success}
-    div.alert.flash.my-2 class="alert-#{flash_mapping.fetch(severity, severity)}" class="alert-#{flash_mapping.fetch(severity, severity)} alert-dismissible fade show"
-      p.mb-0 id="flash-#{severity}" == flash[severity]
-      button.btn-close type="button" data-bs-dismiss="alert" aria-label="Close"
+      - flash_mapping = {alert: :warning, notice: :success}
+      div.alert.flash.mt-5 class="alert-#{flash_mapping.fetch(severity, severity)}" class="alert-#{flash_mapping.fetch(severity, severity)} alert-dismissible fade show"
```

Comment:
Changing the merging here indicates that something with the navbar is not quite right (which I think might be the case -- see my other comment).